frontend/src/modules/EditZing/Components/EditZing.tsx

```diff
@@ -10,10 +11,38 @@ import { GroupGrid } from 'EditZing/Components/GroupGrid'
 import { Student } from 'EditZing/Types/Student'
 import { DndProvider } from 'react-dnd'
 import { HTML5Backend } from 'react-dnd-html5-backend'
+import { getZingGroups, saveSwapStudent } from './Helpers'
+import { FetchedZing } from 'EditZing/Types/Student'
 
 export const EditZing = () => {
+  // get param that was set from history using location
+  const location = useLocation() as any
+  const params = location.state.params
+  const [zingId] = useState(params)
```

Comment:
In which case the code to get the param will just be:

const { search } = useLocation()
const query = new URLSearchParams(search)
const zingId = query.get('id')